src/Charts/chartBar.ts

```diff
@@ -41,20 +42,41 @@ export class ChartBar extends Chart {
     const colorMap = this.getColorMap(colorScale)
     this.group = this.selectGroup(axis, ChartBar.GROUP_CLASS)
 
+    let xRect = (d: unknown, i: number) => { 
+      return i === 0 ? 0 : xScale(data[i-1][xKey])
+    }
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why do we not need this override for `xRect`? Maybe we can define an ESLint ignore pattern for unused variables for e.g. leading underscores.